docs/transpile/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Introduction to the transpiler
+description: Introduction to the transpiler
+
+---
+
+
+# Introduction to the transpiler
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device and optimize the circuit instructions for execution on present-day noisy quantum systems.  
+
+A central component of Qiskit, the transpiler is designed for modularity and extensibility. The goal is to be able to easily write new circuit transformations (known as transpiler **passes**), and combine them with other existing passes.  Which passes are chained together and in which order has a major effect on the final outcome. This pipeline is determined by a **pass manager**, which schedules the passes and also allows passes to communicate with each other by providing a shared space. In this way, the transpiler allows one to begin approaching utility scale work while keeping noise manageable.
+
+This documentation covers the tooling and workflows available to Qiskit users.  If you're using primitives and are only interested in st using the default transpilation options, look to the section [Configure error suppression for Qiskit Runtime.](../run/configure-runtime-compilation)
```

Comment:
"interested in st using" - typo?

Reply:
Possibly my bad -- I recall trying to remove the word just bc it's a potential confuser for translation, and may have only partially removed the word! I'll clean it up

Reply:
done in https://github.com/Qiskit/documentation/pull/260/commits/b7843c5b85e06cf06dd0a17bb930d19f900d5345